edsc/collections-collapsed/src/js/edsc-collections-collapsed.coffee

```diff
@@ -22,6 +22,9 @@ class CollectionsCollapsedModel extends KnockoutComponentModel
     super(params, componentInfo)
     @_scrollParent = dom.scrollParent(@_root)
     @_flyouts = []
+    setTimeout((=>
+      $(window).on 'edsc.overlaychange', @_hideFlyouts), 0)
```

Comment:
This is the only dependency on jQuery in the plugin, so it's not desirable. It also operates globally, when only the control's own flyouts should be hidden